src/pybindings/pycontroller/ControllerBindings/PyChip_Main.cpp

```diff
@@ -0,0 +1,34 @@
+#include <functional>
+#include <map>
+#include <memory>
+#include <stdexcept>
+#include <string>
+
+#include <pybind11/pybind11.h>
+
+typedef std::function<pybind11::module &(std::string const &)> ModuleGetter;
+
+void bind_CHIPController_ChipExceptions(std::function<pybind11::module &(std::string const & namespace_)> & M);
+
+PYBIND11_MODULE(PyChip, root_module)
+{
+    root_module.doc() = "PyChip module";
+
+    std::map<std::string, pybind11::module> modules;
+    ModuleGetter M = [&](std::string const & namespace_) -> pybind11::module & {
+        auto it = modules.find(namespace_);
+        if (it == modules.end())
+            throw std::runtime_error("Attempt to access pybind11::module for namespace " + namespace_ +
```

Comment:
What does this mean - "before it was created" ? It looks like an invalid identifier will trigger the same error?